create-stylelint.mjs

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/* eslint-disable no-process-exit */
+/* eslint-disable no-console */
+'use strict';
+
+const currentVersion = process.versions.node;
+const requiredMajorVersion = parseInt(currentVersion.split('.')[0], 10);
+const minimumMajorVersion = 14;
+
+if (requiredMajorVersion < minimumMajorVersion) {
```

Comment:
[suggest] Checking the running version sounds useful. 👍🏼 
Additionally, by using the `engines.node` field in `package.json`, I believe it is possible to make this check more accurate, though code and dependency (`semver`) increase a bit. For example:

```js
import { fileURLToPath } from 'node:url';
import fs from 'node:fs';
import path from 'node:path';
import semver from 'semver';

const dirname = path.dirname(fileURLToPath(import.meta.url));
const pkg = JSON.parse(fs.readFileSync(path.join(dirname, 'package.json').toString()));

if (!semver.satisfies(currentVersion, pkg.engines.node)) {
	console.error(`Node.js v${currentVersion} is out of date and unsupported!`);
	console.error(`Please use a Node.js version within "${pkg.engines.node}".`);
	process.exit(1);
}
```